libbeat/docs/shared-faq.asciidoc

```diff
@@ -54,6 +54,27 @@ connect to the Lumberjack input plugin. To learn how to install and update
 plugins, see {logstash-ref}/working-with-plugins.html[Working with plugins].
 endif::[]
 
+ifndef::no-output-logstash[]
+[[disconnections-problem]]
+=== {beatname_uc} reports connections being reset by {ls}
```

Comment:
```suggestion
[[publishing-ls-fails-connection-reset-by-peer]]
=== Publishing to {ls} fails with "connection reset by peer" message
```

Thank you for your suggestion. We need to restrict the scope to Logstash (as the Beats protocol use a persistent connection).

Reply:
@dedemorton I adjusted this section to address your previous suggestion. What do you think abou it?

Reply:
LGTM

Reply:
Hope you don't mind...I committed the suggestion so that I can approve the PR.